content/blog/2023-06-07-oreoscan

```diff
@@ -0,0 +1,61 @@
+author: Elena 
```

Comment:
The author name should be lowercase: `elena`, to match the authors in the `authors.ts` file.